package.json

```diff
@@ -122,5 +122,8 @@
     "zeit",
     "css-in-js",
     "css"
-  ]
+  ],
+  "dependencies": {
+    "client-only": "0.0.1"
```

Comment:
Does this need to be a runtime dependency instead of bundling?

Reply:
Yeah it has to be so that if they occurred in the server bundle graph we'll error (https://www.notion.so/vercel/React-Server-Condition-Poisoning-aa31aa23e26d41a18b4450725d37a563)